projects/elements/src/lib/lazy-elements/lazy-element/lazy-element.directive.spec.ts

```diff
@@ -186,4 +206,18 @@ describe('LazyElementDirective', () => {
     );
     expect(appendChildSpy.calls.argsFor(1)[0].type).toBe('module');
   });
+
+  it('uses elementConfig for the tag', () => {
+    const lazyElementsLoaderService = TestBed.get(LazyElementsLoaderService);
+    spyOn(lazyElementsLoaderService, 'getElementConfig').and.returnValue({
```

Comment:
I would rather test it with separate `TestBed` which is pre-configured instead of with service, would remove service reference from here completely to keep it decoupled